server/webhook_integration_test.go

```diff
@@ -0,0 +1,81 @@
+package server
```

Comment:
files are typically named by appending "_test" to the file it is testing. So in this case, I would suggest "webhook_test.go".